generator/main.go

```diff
@@ -86,6 +86,7 @@ func generateConfig(nodes *Node, nameToNode map[string]*Node) {
 }
 
 var (
+	failOnParseErrors  = kingpin.Flag("fail-on-parse-errors", "Exit with a non-zero status if there are parse errors").Default("false").Bool()
```

Comment:
I'd put MIB in here somewhere, as it could be generator.yml you're talking about.

Reply:
Ok.